### DISCUSSION TITLE
When I run as tutorial Note2, I can't get all file (LDASIN.DOMAIN)

### DISCUSSION THREAD

**Discussion by Zhoujieliang** [NCAR/hrldas#137] [Q&A] [answered]
In Note2_2D_NLDAS2_domain.ipynb 3.5.2 (Run the create_forcing.exe) .
After enter './create_forcing.exe namelist.input.NLDAS', there some error. 
And I only get a two file (2000010100.LDASIN_DOMAIN1&HRLDAS_setup_2000010100_d1).
The error is ：
" 
![github](https://github.com/NCAR/hrldas/assets/125653764/db06d056-08e0-4909-bc1d-a686de79bcd6)
Source model land/sea data not found; assuming a bitmap exists
 You have this many undefined soil points over valid land points:          212
 This tends to happen around coastlines or for unresolved islands in your forcin
 g data
 ===== Consider increasing expand_loop in namelist =====
 However, this may also occur if your forcing source does not cover your domain.
"

**Comment by cenlinhe** [comment, marked as answer]
please increase the EXPAND_LOOP to a larger number (e.g., 5 or 10) in namelist.input.NLDAS. This error occurs because there are some unresolved island points or coastal grids during spatial interpolation. Increasing EXPAND_LOOP will help resolve those points.

**Reply by Zhoujieliang** [reply]
Thank you!! I can run it!  Can you introduce me to any good methods for displaying output files（LDASOUT_DOMAIN）, reading or visualizing them.   Especially if I want to use latitude and longitude points to read the values of the output file

**Reply by cenlinhe** [reply]
You can use ncview to directly view the netcdf file (the output *LDASOUT_DOMAIN files are netcdf files) or use python/matlab/IDL to write the code to read in the output files.
